### DISCUSSION TITLE
Run an AfterDark screen saver (specifically, Stained Glass)?

### DISCUSSION THREAD

**Discussion by emendelson** [otya128/winevdm#1305] [Q&A]
Is there any way to run the AfterDark screen savers on modern Windows? The one I'm most interested in is Stained Glass. Thanks for any help.

**Comment by cracyc** [comment]
In windows 10 at least if you install it then go to settings and search for screen saver it can be enabled there.  I've done this with stand alone screen savers but ought to work with after dark too.

**Comment by emendelson** [comment]
After Dark seems to work differently from standard screen savers. I've now figured out that I could install it, and launch ADTRAY.EXE from C:\AFTERDRK. This program lets me choose among any .AD screen savers that I've saved in the same folder. The Setup button lets me specify a hot corner that turns on the screen saver. 

I don't think there's a way to launch this as an ordinary screen saver, but this is more than good enough. I'm amazed and impressed to have this working again after so many years. Thank you!

**Comment by cracyc** [comment]
It predates the windows 3.1 screen saver support i guess.  That interface is still supported in windows 10.

**Comment by DaVyper** [comment]
i gotta look into this as i'd love to get my old "loony toons"/"totally twisted" ad's running again
